### ISSUE TITLE
bad performace on PCB and AlignedReID++.

### ISSUE THREAD

**Issue by magicffourier** [michuanhaohao/reid-strong-baseline#127]
@michuanhaohao I also try PCB and AlignedReID++.
In the paper, you said

> Our baseline boosts the performance of k-reciprocal reranking, PCB, AligedReID++, and CamStyle by a large margin

I found you learning rate is not similar with PCB and AlignedReID++. And I also get bad performance .
Could you please explain it.

_Originally posted by @magicffourier in https://github.com/michuanhaohao/reid-strong-baseline/issues/60#issuecomment-568731157_

**Comment by michuanhaohao** [comment]
Because we trained models with some different tricks, learning rate was different from other works.

We reproduced Alignedreid++ and PCB on our baseline. All training settings were similar with our strong baseline. You can try it again.

**Comment by magicffourier** [comment]
I find that your strong baseline and AlignedReid++ have different triplet sampler. Which one should I use?

**Comment by magicffourier** [comment]
I use the same triplet sampler, lr and same batchsize as this repo. And I only get 85.3/74.8 for Rank1 and mAP respectively

**Comment by michuanhaohao** [comment]
Using the old triplet sample, I (Alignedreid++) only achieved 82.1/69.7 for Rank1 and mAP respectively on DukeMTMC-ReID. Thus, you achieved better performance with these tricks. Did you combine the local distance with global distance?

**Comment by magicffourier** [comment]
I combine the local distance with global distance. And I notice bnnneck mainly improve the performance of global feature because you have already applied bn->relu->l2 norm to locaal feature. And I guess the improvement of local feature might be related to the improvement of global feature
